### DISCUSSION TITLE
Multiphase simulation

### DISCUSSION THREAD

**Discussion by Keivou** [ProjectPhysX/FluidX3D#110] [Q&A] [answered]
Hey, long story short, I need to simulate the interaction between water and oil in a tank, but for the life of me I cannot find a way to make it so that there are two types of fluid flowing on the same sim box. Can this be done? I was thinking of the alternative of using particles and giving them oil-like features, but hard to tell to what extent could that be considered representative for oil interaction.

**Comment by ProjectPhysX** [comment, marked as answer]
Multiphase is not supported. The SURFACE extension also is only single-phase, as the gas phase is not modeled at all. It's not possible to use particles for multi-phase either, as they cannot modify viscosity nor model the meniscus.

**Reply by Keivou** [reply]
I still had a bit of hope. Your fast response saves me a loooot of time, I also learned a TON in the few days that I've been using your creation, which will definitely become useful. I'll definitely keep using this for single phase testing :)
